tests/test_35_oidc_token_endpoint.py

```diff
@@ -149,169 +173,125 @@ def create_endpoint(self, conf):
             "response_types": ["code", "token", "code id_token", "id_token"],
         }
         endpoint_context.keyjar.import_jwks(CLIENT_KEYJAR.export_jwks(), "client_1")
+        self.session_manager = endpoint_context.session_manager
         self.endpoint = endpoint_context.endpoint["token"]
+        self.user_id = "diana"
 
     def test_init(self):
         assert self.endpoint
 
-    @pytest.mark.parametrize("grant_types_supported", [
-        {
-            "authorization_code": {
-                "class": AccessToken
-            },
-            "refresh_token": {
-                "class": RefreshToken,
-            },
-        },
-        {},  # Empty dict should work with the default grant types
-        {
-            "authorization_code": {
-                "class": "oidcendpoint.oidc.token_coop.AccessToken"
-            },
-        },
-        {
-            "authorization_code": {
-                "class": "oidcendpoint.oidc.token_coop.AccessToken",
-                "kwargs": {},
-            },
-        },
-        {
-            "authorization_code": True,  # Both True and None end up using the defaults
```

Comment:
These tests were removed and would catch the bug I reviewd above relating to the initialization of the helper endpoint classes.